frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -545,4 +563,34 @@ export function applyState(state: PartialUrlManagerState, ignoreZoom: boolean =
   if (state.rotation != null) {
     Store.dispatch(setRotationAction(state.rotation));
   }
+  if (state.activeMappingByLayer != null) {
+    for (const layerName of Object.keys(state.activeMappingByLayer)) {
+      const { mappingName, mappingType, agglomerateIdsToImport } = state.activeMappingByLayer[
+        layerName
+      ];
+
+      let effectiveLayerName;
+      try {
+        const { dataset } = Store.getState();
+        // The name of the layer could have changed if a volume tracing was created from a viewed annotation
+        effectiveLayerName = getSegmentationLayerByNameOrFallbackName(dataset, layerName).name;
+      } catch (e) {
+        console.error(e);
+        // TODO: Proper error message
+        Toast.error(e);
+        continue;
+      }
+
+      message.loading({ content: "Activating Mapping", key: MAPPING_MESSAGE_KEY });
+      Store.dispatch(setMappingAction(effectiveLayerName, mappingName, mappingType));
+
+      if (mappingType === "HDF5" && agglomerateIdsToImport != null) {
```

Comment:
`agglomerateIdsToImport` can only be hand-crafted, right?

Reply:
Yes, for now. If we ever decide to keep track of the loaded agglomerate Ids, we could also persist that in the sharing links.